src/SIS_slow_thermo.F90

```diff
@@ -857,6 +854,11 @@ subroutine SIS2_thermodynamics(IST, dt_slow, CS, OSS, FIA, IOF, G, US, IG)
     IOF%lprec_ocn_top(i,j) = IOF%lprec_ocn_top(i,j) + IST%part_size(i,j,k) * FIA%lprec_top(i,j,k)
   enddo ; enddo ; enddo
 
+  ! Reset sea ice melt rate to zero
+  do j=jsc,jec ; do i=isc,iec
+    IOF%seaice_melt(i,j) = 0.
```

Comment:
does this need to be a loop or would `IOF%seaice_melt(:,:)=0.` work just as well?

Reply:
I think you're probably right. I can test it.

Reply:
Fixed in https://github.com/NOAA-GFDL/SIS2/pull/214/commits/49a2829d4f9f6f1024b119eb4a35c1398e935f14